package.json

```diff
@@ -11,9 +11,18 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
```

Comment:
why are we adding `@fortawesome` dependency?

Reply:
for easy icons

Reply:
i don't think we need a package to manage icons